src/components/calcite-checkbox/readme.md

```diff
@@ -28,13 +28,14 @@ If you don't pass in an input, calcite-checkbox will act as the source of truth:
 | `name`          | `name`          | The name of the checkbox input                                                                                                                | `string`            | `""`        |
 | `scale`         | `scale`         | specify the scale of the checkbox, defaults to m                                                                                              | `"l" \| "m" \| "s"` | `"m"`       |
 | `theme`         | `theme`         | Determines what theme to use                                                                                                                  | `"dark" \| "light"` | `undefined` |
-| `value`         | `value`         | The value of the checkbox input                                                                                                               | `string`            | `""`        |
+| `value`         | `value`         | The value of the checkbox input                                                                                                               | `string`            | `undefined` |
 
 ## Events
 
-| Event                   | Description                                      | Type               |
-| ----------------------- | ------------------------------------------------ | ------------------ |
-| `calciteCheckboxChange` | Emitted when the checkbox checked status changes | `CustomEvent<any>` |
+| Event                          | Description                                      | Type               |
+| ------------------------------ | ------------------------------------------------ | ------------------ |
+| `calciteCheckboxChange`        | Emitted when the checkbox checked status changes | `CustomEvent<any>` |
+| `calciteCheckboxFocusedChange` | Emitted when the checkbox focused state changes  | `CustomEvent<any>` |
```

Comment:
Unrelated, but this is a weird event. Why not `calciteCheckboxFocus` and `calciteCheckboxBlur`? 

The `focused` property is odd too. Seems like a method would be better for handling that since the state of being focused can be changed outside of the component by the user and this property would not get updated.

Reply:
Yeah, agree - having `focused` is a weird prop and we could just add a `setFocus` method that we've called most others - can open an issue for those? These were just stray readme updates that never made it in past PR.

Reply:
Follow up - https://github.com/Esri/calcite-components/issues/808